add-constraint.md

```diff
@@ -51,11 +51,66 @@ Adding the [Check constraint](check.html) requires that all of a column's values
 
 ### Add the Foreign Key Constraint
 
-Adding the [Foreign Key constraint](foreign-key.html) requires that all of a column's values are equal to an existing value in the column it references.
-
-``` sql
-> ALTER TABLE orders ADD CONSTRAINT customer_fk FOREIGN KEY (customer) REFERENCEs customers (id);
-```
+To add the [Foreign Key](foreign-key.html) constraint to an existing column or group of columns, you must first use [`CREATE INDEX`](create-index.html) to index the column or columns that you plan to constrain. Then you can use the `ADD CONSTRAINT` statement to add the Foreign Key constraint to the column or columns.
```

Comment:
to clarify, only need to create index if there isn't already an index that contains the column(s).